src/sonic-framework/rebootbackend/rebootbackend.cpp

```diff
@@ -0,0 +1,12 @@
+#include "interfaces.h"
+#include "reboot_interfaces.h"
+#include "rebootbe.h"
+
+using namespace ::rebootbackend;
```

Comment:
using namespace in global scope is generally considered bad practice. We should either do `using ::rebootbackend::Foo` or type out the entire namespace.

Reply:
Pl. refer to https://github.com/sonic-net/sonic-buildimage/pull/20713. I have updated as per review comment.